README.md

```diff
@@ -1,2 +1,30 @@
 # Bitcoin Design - Meta
-`community`, `project-management`, `process`
\ No newline at end of file
+`community`, `project-management`, `process`
+
+Anything related to group consensus and coordination of the Bitcoin Design Community.
+
+### Getting Started
+1. Pick existing [project](https://docs.google.com/document/d/1P20rPq6V6w78dXpBqLiiiB-kSypZOzXaG9ZUErYIq28/edit) in the space and do something small
```

Comment:
My understanding of our process is that we'll use GitHub Projects, so the link should point to the list of active projects on GitHub not a Google doc